### PR TITLE
add to cart

### PR DESCRIPTION
**Pull request by JeffNewcomer**
screen shot 2018-07-03 at 10 27 50 am

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -236,21 +236,30 @@ function createFullDetailsCard(item) {
 
   var $buttonContainer = document.createElement('div')
   $largeCardTextContainer.appendChild($buttonContainer)
+  // $buttonContainer.style.border = 'red solid 5px'
```
Comment (thebearingedge) [inline main.js]: Please leave out commented code.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
After you have addressed the commented code, please include a `.gif` of your working feature.

**Comment by JeffNewcomer** [comment]
Fixed and committed.

**Comment by JeffNewcomer** [comment]
jamazon

**Comment by thebearingedge** [comment]
Looks like you've got a bug causing the details of the previously viewed item still in the `data-view="details"`. Once that's fixed we can merge this.

**Pull request closed without merging**